libraries/AP_FlashStorage/AP_FlashStorage.cpp

```diff
@@ -286,6 +308,9 @@ bool AP_FlashStorage::load_sector(uint8_t sector)
             // invalid state
             return false;
         }
+#if AP_FLASHSTORAGE_TYPE == AP_FLASHSTORAGE_TYPE_H7
+        ofs = (ofs + 31U) & ~31U;
```

Comment:
& -31 ???

This, at the very least, needs a comment

Reply:
comment added